subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_location

```diff
@@ -8,3 +8,18 @@ config NRF_CLOUD_LOCATION
 	imply FPU
 	depends on NRF_CLOUD_MQTT
 	select CJSON_LIB
+
+config NRF_CLOUD_LOCATION_ANCHOR_LIST
+	bool "Include the list of anchor names when receiving location results from the callback"
+	depends on NRF_CLOUD_LOCATION
+	depends on WIFI
+
+config NRF_CLOUD_LOCATION_ANCHOR_LIST_BUFFER_SIZE
+	int "Size of the buffer to contain the anchor name list"
+	depends on NRF_CLOUD_LOCATION_ANCHOR_LIST
+	range 37 702
+	default 142
```

Comment:
These are rather specific numbers. I'm curious where they come from.

Reply:
the size of a JSON array with one anchor = 37.
`["<32_chars>"]<NULL>`